eunji/src/App.tsx

```diff
@@ -0,0 +1,40 @@
+import FnB from './components/FnB';
+import Introduce from './components/Introduce';
+import Description1 from './components/Description1';
+import Description2 from './components/Description2';
+
+import styled from 'styled-components';
+
+function App() {
+  return (
+    <Container>
+      <Introduce />
+      <StyledDiv>
+        <FnB name="교다이야" location="합정" img="fnb1.jpg" menu="가케우동정식(\13,000)" review="https://www.siksinhot.com/P/365707"></FnB>
+        <Description1 />
+      </StyledDiv>
+      <StyledDiv>
+        <FnB name="훈닭포차" location="까치산" img="fnb2.jpg" menu="숯불바베큐치킨(\32,000)" review="https://www.siksinhot.com/P/868189"></FnB>
```

Comment:
```suggestion
        <FnB name="훈닭포차" location="까치산" img="fnb2.jpg" menu="숯불바베큐치킨(\32,000)" review="https://www.siksinhot.com/P/868189" />
```
로 하는 게 더 권장되는 사항입니다 ~~

Reply:
피드백 반영해서 커밋 완료했습니다!